Converters/Civil3d/Speckle.Converters.Civil3dShared/ToSpeckle/Raw/AlignmentSubentityArcToSpeckleRawConverter.cs

```diff
@@ -22,37 +21,30 @@ IConverterSettingsStore<Civil3dConversionSettings> settingsStore
 
   public SOG.Arc Convert(CDB.AlignmentSubEntityArc target)
   {
+    // alignment arcs do not have the same properties as autocad arcs.
+    // we're assuming they are always 2d arcs on the xy plane to calculate the midpoint
```

Comment:
Is this assumption correct, or is there a chance a civil arc won't be laying on the 2d plane?

Reply:
They appear to always be of type `Arc2d`